pages/index.tsx

```diff
@@ -1,6 +1,7 @@
 'use client';
```

Comment:
I think we don't want to render whole pages on the client side, do we?

Reply:
yes